homeassistant/generated/integrations.json

```diff
@@ -2873,6 +2873,10 @@
       "config_flow": true,
       "iot_class": "cloud_polling"
     },
+    "lawn_mower": {
```

Comment:
I don't think this should be added. There's probably some exception in hassfest we should update.

Reply:
Yeah ended up auto added